pkg/manager/cluster.go

```diff
@@ -99,10 +99,38 @@ func (c *Cluster) generateSupportBundleYAMLs(yamlsDir string, errLog io.Writer)
 
 		done[namespace] = struct{}{}
 	}
+	for _, collector := range c.sbm.ExtraBundleCollectors {
+		switch collector {
+		case "Harvester":
```

Comment:
Do we want to use `strings.ToLower(collector)` and check the value `harvester`? Or do we want it to be exactly `Harvester`?

Reply:
good catch, it should. That we could handle both `harvester` and `Harvester` from config.

Reply:
fixed